addons/nuxeo-drive/elements/nuxeo-drive-edit-button.js

```diff
@@ -0,0 +1,106 @@
+/**
+(C) Copyright 2016 Nuxeo SA (http://nuxeo.com/) and others.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+Contributors:
+  Nelson Silva <nsilva@nuxeo.com>
+*/
+import { html } from '@polymer/polymer/lib/utils/html-tag.js';
+import { I18nBehavior } from '@nuxeo/nuxeo-ui-elements/nuxeo-i18n-behavior.js';
+import { FiltersBehavior } from '@nuxeo/nuxeo-ui-elements/nuxeo-filters-behavior.js';
+
+/**
+`nuxeo-drive-edit-button`
+@group Nuxeo UI
+@element nuxeo-drive-edit-button
+*/
+Polymer({
+  _template: html`
+    <style include="nuxeo-action-button-styles"></style>
+
+    <nuxeo-resource id="token" path="/token" params="{&quot;application&quot;: &quot;Nuxeo Drive&quot;}"></nuxeo-resource>
```

Comment:
inline JSON is screwed up?
```suggestion
    <nuxeo-resource id="token" path="/token" params='{"application": "Nuxeo Drive"}'></nuxeo-resource>
```